pkg/rtc/transport.go

```diff
@@ -1330,7 +1371,9 @@ func (t *PCTransport) handleRemoteICECandidate(e *event) error {
 		return nil
 	}
 
+	t.lock.Lock()
 	t.allowedRemoteCandidates = append(t.allowedRemoteCandidates, c.Candidate)
+	t.lock.Unlock()
```

Comment:
Sorry, yeah this needs to be moved above the pending check and return. Just moving this block above the previous block should be good.
Till now, the allowed/filtered candidates fields were purely informational (only used for logging all candidates in one log). So, it is not critical. But, to be correct, this should be moved above the previous block.

Reply:
got it, will move it.